src/main/java/pl/wrapper/parking/pwrResponseHandler/dto/ParkingResponse.java

```diff
@@ -12,20 +11,15 @@ public record ParkingResponse(
         int parkingId,
         int freeSpots,
         int totalSpots,
-        @Schema(example = "best parking")
-        String name,
-        @Schema(example = "WRO")
-        String symbol,
-        @Schema(type = "string", format = "time", example = "08:00:00")
-        @Nullable LocalTime openingHours,
-        @Schema(type = "string", format = "time", example = "22:00:00")
-        @Nullable LocalTime closingHours,
```

Comment:
Why are we changing the code formatting here and in several other places? Putting everything in one line makes it less readable. Let's revert the formatting changes here and wherever needed